packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -180,8 +190,10 @@ export class MenusContributionPointHandler {
         const command: Command = { id };
         this.commands.registerCommand(command, handler);
 
-        const { group, when } = action;
-        const item: Mutable<TabBarToolbarItem> = { id, command: id, group: group || '', when };
+        const { when } = action;
+        const [group, sort] = (action.group || 'navigation').split('@');
```

Comment:
`action.group || 'navigation'` looks bogus if `action.group` has whitespace characters

Could you preserve changes from the original PR?

Reply:
done